tests/integrationv2/test_well_known_endpoints.py

```diff
@@ -90,6 +90,7 @@
 @pytest.mark.parametrize("endpoint", ENDPOINTS, ids=get_parameter_name)
 @pytest.mark.parametrize("provider", [S2N], ids=get_parameter_name)
 @pytest.mark.parametrize("cipher", CIPHERS, ids=get_parameter_name)
+@flaky_test()
```

Comment:
this can't use `@pytest.mark.flaky(reruns=5)`?

Reply:
we already use it elsewhere so I assume its setup: https://github.com/aws/s2n-tls/blob/87ef08339a372987802154986e018004f00a710c/tests/integrationv2/test_key_update.py#L20

Reply:
Looks like [this package](https://github.com/pytest-dev/pytest-rerunfailures/tree/master) implements the plugin. But I don't see support for incremental backoff, which I do think is important for this kind of test.

Reply:
Hmm I think exponential backoff would be necessary if we are worried about bringing down a dependency, but I wouldn't expect that in this case :-) Do you think just the static `reruns-delay` would be sufficient?

Reply:
I guess we are pulling in the plugin, too: https://github.com/aws/s2n-tls/blob/87ef08339a372987802154986e018004f00a710c/tests/integrationv2/tox.ini#L15

I guess I can just make it have a longer delay between runs and call it good